apps/gp2-server/src/data-providers/working-group.data-provider.ts

```diff
@@ -55,14 +69,30 @@ export class WorkingGroupSquidexDataProvider
     };
   }
 
+  async update(
+    id: string,
+    workingGroup: gp2Model.WorkingGroupUpdateDataObject,
+  ): Promise<void> {
+    const squidexWorkingGroup = convertToSquidexWorkingGroup(workingGroup);
+
+    await this.squidexRestClient.patch(id, squidexWorkingGroup);
+  }
   private async queryFetchByIdData(id: string) {
-    return this.squidexGraphlClient.request<
+    return this.squidexGraphqlClient.request<
       FetchWorkingGroupQuery,
       FetchWorkingGroupQueryVariables
     >(FETCH_WORKING_GROUP, { id });
   }
 }
 
+const convertToSquidexWorkingGroup = (
+  workingGroup: gp2Model.WorkingGroupUpdateDataObject,
+) =>
```

Comment:
I guess it would be useful to declare the output as `gp2Squidex.InputWorkingGroup["data"]`?

Reply:
done